Adding webdfs java client upload example (#5)
cloudmelon/sql-server-samples: samples/features/sql-big-data-cluster/connectivity/webhdfs-java-client/README.md

> # Building this example in Linux

Reviewer: ```suggestion
# Building this example on Linux
```

## Prepare dependencies jar into work dir

bash ./prepare.sh

> ## Building jars

Reviewer: Newline before this statement.

bash ./build.sh

# Running this sample

> The App.java has hardcoded values in it mentioned in the sections below. Those should be pointed to the right values.

Reviewer: It might be helpful to parameterize the hardcoded values.

Reviewer 2: Agreed ( including the file name mentioned in one of the comments ), thanks

> Before running this application, the kinit which somes with the JRE needs to be run to get a kerberos token.

Reviewer: ```suggestion
On Windows, before running this application, kinit needs to be run to get a kerberos token. `kinit` is shipped with the JRE for Windows Java distributions.
```

`<JRE_INSTALL_ROOT>\bin\kinit.exe <user>@<REALM>`

e.g. 

`<JRE_INSTALL_ROOT>\bin\kinit.exe admin@ARIS.LOCAL`

> In Linux, you can directly using kinit in system.

Reviewer: ```suggestion
On Linux, kinit can be used. If `kinit` is not installed, then it would need to be installed to get a kerberos token.
```

e.g.

 kinit admin@ARIS.LOCAL

# Sample login.conf. There is an existing login.conf that is being checked in as well.

For more details about login.conf refer https://docs.oracle.com/javase/7/docs/technotes/guides/security/jgss/tutorials/LoginConfigFile.html

```
com.sun.security.jgss.login {

>     com.sun.security.auth.module.Krb5LoginModule required client=TRUE useTicketCache=true debug=true;

Reviewer: In this markdown, the login.conf should show that the principal="admin" can be specified in the login.conf itself.
